docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -346,7 +346,7 @@ here overrides that default:
 
   proxy_servers:
       http: http://user:pass@corp.com:8080
-      https: https://user:pass@corp.com:8080
+      https: http://user:pass@corp.com:8080
```

Comment:
````suggestion
      https: http://user:pass@corp.com:8080
      
```{admonition} Mixing HTTPS and HTTP
The protocol in the URL (either ``http://`` or ``https://``) should match the actual protocol of your proxy server. The keys ``http`` and ``https`` merely indicate the type of traffic to route, not the protocol of the proxy server itself. Ensure that both keys use the correct protocol based on your proxy server's configuration.
```
````